.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: ruby
+rvm:
+  - 1.8.7
```

Comment:
Here is where I am requiring a version, for this older branch of say_when, to keep working on 1.8.7 since that is what prx.org is using.